libraries/Microsoft.Bot.StreamingExtensions/SerializationSettings.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    /// <summary>
+    /// This class defines the settings used when serializing data contained by objects
+    /// included as part of the Bot Framework Protocol v3 with Streaming Extensions.
+    /// </summary>
+    public static class SerializationSettings
+    {
```

Comment:
Why do we have so many settings? Can't we just have 1? I think we already do... :)

Reply:
These mirror the ones used by the Azure Bot Service and Bot Builder, but as this is not dependent on those, it must be redefined.